src/js/views.js

```diff
@@ -52,6 +71,51 @@ window.MapView = Backbone.View.extend({
             this.map.invalidateSize(animate);
         }
     },
+
+    updateMarkers: function() {
+        var subregions = null;
+
+        if (this.model.get('selectedRegion')) {
+            subregions = this.model.get('selectedRegion').get('subregions');
+
+            if (subregions && subregions.length > 0) {
+                _.forEach(subregions, function(region) {
+                    if (region.mapCenter) {
+                        this.markers.addLayer(this.createMarker(region));
+                    }
```

Comment:
Wonder if it's worth adding an `else` here to `console.warn("Region's `mapCenter` is not configured properly.")`.

Reply:
I don't think it's worth it at the moment. I'd say we give them a chance to figure out the configuration, and if there are any issues, go from there.